### ISSUE TITLE
Render ClickHouse addon application fail

### ISSUE THREAD

**Issue by JingbenShi668** [kubevela/catalog#511]
When I enabled clickhouse with vela in k8s cluster, I don't know why the  clickhouse addon rendered the application fail.
<img width="1522" alt="image" src="https://user-images.githubusercontent.com/39040502/197341301-6b7495b1-7051-4694-b1a3-fc7efc8ca4a7.png">

Besides, the clickhouse addon is an experimental addon, why it appears in the VelaUX as a verified addons?😂
<img width="829" alt="image" src="https://user-images.githubusercontent.com/39040502/197341381-314d6c37-4543-4b57-aa83-6d0459f44868.png">

**Comment by charlie0129** [comment]
Please provide your KubeVela version. Did you add the experimental registry? If you did, yes it will show up.

**Comment by charlie0129** [comment]
Currently, the ClickHouse addon uses a feature that is only available in KubeVela v1.6-alpha.1 and later. So, to use the latest version of ClickHouse addon, a new version of KubeVela is required. Or you can use older versions.

We will fix this soon by adding a vela version requirement.

**Comment by JingbenShi668** [comment]
> Please provide your KubeVela version. Did you add the experimental registry? If you did, yes it will show up.

Yes, I have add the  the experimental registry as addon repo. And, my KubeVela Version is 1.5+, I think I have install the latest KubeVela. 😂 
It maybe is that the version of my KubeVela is 1.5+ lower than v1.6-alpha.1 and it lead to that I can't normally enable clickhouse 1.0.1 addon . 
<img width="374" alt="image" src="https://user-images.githubusercontent.com/39040502/197358951-abbaf481-b2ab-4b3a-bb17-b471bd9beb65.png">
<img width="670" alt="image" src="https://user-images.githubusercontent.com/39040502/197358994-96e7265d-170f-4022-8e05-84b1d34891d3.png">

**Comment by JingbenShi668** [comment]
> Currently, the ClickHouse addon uses a feature that is only available in KubeVela v1.6-alpha.1 and later. So, to use the latest version of ClickHouse addon, a new version of KubeVela is required. Or you can use older clickhouse addon versions.
> 
> We will fix this soon by adding a vela version requirement.
When I enabled the clickhouse 1.0.0 with KubeVela 1.5.7,  it seemed addon clickhouse is enabled successfully.
<img width="1418" alt="image" src="https://user-images.githubusercontent.com/39040502/197359333-5f4f73f5-0efa-40ae-801c-ff2dd40e53df.png">

After I applied the clickhouse app, the application prod/ck-app  were applied. However, when I checked the status pf the application ck-app, I don't understand what errors resulted in the failure of the applicaion applying.
<img width="1516" alt="image" src="https://user-images.githubusercontent.com/39040502/197359416-83342267-81f5-4fb0-9e0e-999b8480757e.png">

**Comment by charlie0129** [comment]
Looks like your ingress controller is having some problems. For testing purposes, try removing Ingress or gateway trait in your app.
